docs/release-info/upgrade-notes.md

```diff
@@ -193,6 +193,34 @@ Removed the `auto` search strategy from the native search query. Setting `search
 
 [#15550](https://github.com/apache/druid/pull/15550)
 
+### Incompatible changes
+
+#### Changes to date format in kubernetes service discovery
+
+Druid 29.0.0 includes a breaking change for those using kubernetes service discovery. The date format used in the config maps used for leader election of coordinators and overlords has changed. 
```

Comment:
Please describe the error a little more or maybe include an example of the encountered error message.